### PR TITLE
Proper update to Julia 1.0 and fix tests. 

### PR DESCRIPTION
**Pull request by 1ozturkbe**
I fixed the deprecations and hacked the lexer issues. Couldn't find exactly why the lexer was having trouble with certain relations, bt as far as I can tell, full functionality has been restored. @timholy please review and merge to get this repo back in shape!

### CODE REVIEW

**Comment by 1ozturkbe** [comment]
@timholy also for the future, I would appreciate it if you could add me as a collaborator to this repo so I can have broader access to make fixes.

File: src/parser.jl [hunk 1]
```diff
@@ -264,6 +264,7 @@ function parseexprs(lexed, debug=false)
         if length(thisarg) != 1
             println(stderr, "thisarg:")
             foreach(x->(print(stderr, "  "); sexpr(stderr, x); println(stderr)), thisarg)
+            print(length(thisarg))
```
Comment (timholy) [inline src/parser.jl]: Intentional?

**Review by timholy — COMMENTED** [review]
Squash-merge whenever you are ready!

**Comment by timholy** [comment]
I can also transfer ownership to some organization if you prefer. (I'm not sure there's a way to transfer ownership to an individual, though I haven't tried.)

File: src/parser.jl (hunk 1 above)
Comment (1ozturkbe) [reply src/parser.jl]: Good catch! The block is useful but the line is definitely not needed. Will fix rn.

**Comment by 1ozturkbe** [comment]
@timholy if you no longer want to deal with this repo, you can transfer ownership to me (1ozturkbe) under Settings/Options, and scrolling to the bottom to the "Danger Zone".  As of rn though, I do not have write access.

**Comment by timholy** [comment]
I know about transfers, I just wasn't sure whether I'd need write permissions in your account to do the transfer. With orgs you do.

I'd prefer to transfer it to you. Currently your fork gets in the way, though.

**Comment by 1ozturkbe** [comment]
Alright, I deleted the fork. It should be set now.

**Pull request merged**